diff --git a/src/Message/SmsNotification.php b/src/Message/SmsNotification.php
index 192aa09..1bdfe34 100644
--- a/src/Message/SmsNotification.php
+++ b/src/Message/SmsNotification.php
@@ -10,4 +10,9 @@ class SmsNotification
     {
         $this->content = $content;
     }
+
+    public function getContent(): string
+    {
+        return $this->content;
+    }
 }
diff --git a/src/MessageHandler/SmsNotificationHandler.php b/src/MessageHandler/SmsNotificationHandler.php
new file mode 100644
index 0000000..8c340cf
--- /dev/null
+++ b/src/MessageHandler/SmsNotificationHandler.php
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\MessageHandler;
+
+use App\Message\SmsNotification;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+class SmsNotificationHandler implements MessageHandlerInterface
+{
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    public function __invoke(SmsNotification $notification)
+    {
+        $this->logger->info($notification->getContent());
+    }
+}
